protos/bottle/assembly/v1/events.proto

```diff
@@ -9,5 +9,6 @@ message BuildCreated {
 }
 
 message BuildUpdated {
-  bottle.assembly.v1.Build build = 1;
+  bottle.assembly.v1.Build old = 1;
+  bottle.assembly.v1.Build new = 2;
```

Comment:
This is a breaking change but we don't currently use this message. This change updates the message to follow our new format for update messages.